### ISSUE TITLE
Make QueueAttribute public instead of sealed

### ISSUE THREAD

**Issue by gavinharriss** [HangfireIO/Hangfire#503]
I'd like to set the queue from configuration rather than a static string. In order to do this I must inherit from QueueAttribute and set the Queue property in code in order to pull the name in from my configuration app settings.

Much like is described here:
http://stackoverflow.com/questions/18619184/using-config-settings-in-attributes

Unfortunately QueueAttribute is sealed so I can't use this approach. Is it possible to make this public?

BTW: Hangfire is an awesome tool!

**Comment by gandarez** [comment]
If you fork the repo and simply unseal the class you'll achieve what you need.

**Comment by gavinharriss** [comment]
I've simply copied https://github.com/HangfireIO/Hangfire/issues/503 into my own code base for now, renamed it and tweaked the functionality. I figured inheriting directly (via having an unsealed class) would cause less issues in the future when upgrading to newer versions of Hangfire via Nuget. Also figured it would be useful for other people in same situation making solution more developer friendly.
